deploy/catalogsourceconfig.cr.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   name: "example"
 spec:
   targetNamespace: "global"
+  packages: "operators/example"
```

Comment:
In the future, It would be easier to handle `packages` in a more structured manner -
```yaml
name:         opsrc-global-operators
namespace:    kube-system
spec:
  packages: 
     - registryNamespace: operators
       repository: certified 
       release: 1.0.0 ## extensible, more attributes can be added  
     - registryNamespace: operators
       repository: redhat 
       release: 1.2.0
  targetNamespace:  kube-system
```

Reply:
Sure. I would prefer doing that in a separate PR once we have nailed down what the external data-source actually is and how it's layout is configured.

Reply:
That's my expectation as well.